backend/config.js

```diff
@@ -21,14 +14,34 @@ const config = {
     },
   },
   geoService: {
-    host: process.env.GEO_SERVICE_URL,
+    host: `http://${process.env.GEO_SERVICE_URL}`,
```

Comment:
This environment variable probably shouldn't be called `URL` if you prefix it with `http://` and actually use `hostname` as the environemtn variable
So either use hostname or use a real URL, maybe better consistent with the other settings